test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -324,6 +326,80 @@ async function tests() {
 
     assert.strictEqual(data, expected);
   })();
+
+  await (async function() {
+    console.log('.next() on destroyed stream');
```

Comment:
Are the console.log statements part of the test? If not, perhaps make them comments instead?

Reply:
The whole file is structured this way. These are very useful for debugging.